hack/lib/etcd.sh

```diff
@@ -28,7 +28,15 @@ kube::etcd::validate() {
   }
 
   # validate it is not running
-  if pgrep -x etcd >/dev/null 2>&1; then
+  case "$(uname -s)" in
+    Linux)
+      etcd_running="pgrep --ns $$ --nslist net -x etcd"
```

Comment:
Well these scripts are just using for testing. I don't think we need to do like this.

@ixdy @mkumatag FYI. PTAL.

Reply:
You mean the check for `Linux` or the change in general?

Right now I have to stop etcd instances in other net namespaces for no good reason before I can test, which is inconvenient.